notebooks/tel_and_site/subsys_req_ver/tma/SITCOM-1120_az_brake_analysis.ipynb

```diff
@@ -0,0 +1,1197 @@
+{
```

Comment:
<p>Adding a quick one line description as the following might be helpful:</p><blockquote>A function to convert input timestamp into iso format "YYYY-MM-DDTHH:MM:SS.SSS". Input values are start time and end time for a period. </blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/120/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Description:</p><p>The <code>format_time</code> function converts two input timestamps (<code>t_start</code> and <code>t_end</code>) from ISO 8601 format to UTC scale using the <code>Time</code> class. It returns a tuple.</p>